### PR TITLE
Bug 1166270 - Replace the "confirm" dialog for SMS STK commands by an…

### PR DESCRIPTION
**Pull request by julienw**
… alert r=timdream

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=21becbc0e5074ea5ba883f65dd7be9593325e307

File: apps/system/test/unit/icc_worker_test.js [hunk 1]
```diff
@@ -1,20 +1,19 @@
 'use strict';
 
-/* global MocksHelper, MockNavigatorMozIccManager, MockSystemICC, icc_worker,
-          MockNotificationHelper */
+/* global MocksHelper, MockNavigatorMozIccManager, MockSystemICC,
+          icc_worker, icc,
+          MockNotificationHelper, STKHelper */
 
-require('/shared/test/unit/mocks/mock_l10n.js');
```
Comment (julienw) [inline apps/system/test/unit/icc_worker_test.js]: looks like it was useless in the test

File: apps/system/test/unit/icc_worker_test.js [hunk 2]
```diff
@@ -1,20 +1,19 @@
...
 require('/shared/test/unit/mocks/mock_navigator_moz_icc_manager.js');
 require('/shared/test/unit/mocks/mock_notification_helper.js');
 require('/shared/test/unit/mocks/mock_dump.js');
 require('/shared/test/unit/mocks/mock_stk_helper.js');
-requireApp('system/js/icc_worker.js');
+require('/js/icc_worker.js');
 
 var mocksForIcc = new MocksHelper([
   'Service',
-  'L10n',
```
Comment (julienw) [inline apps/system/test/unit/icc_worker_test.js]: same

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=2bf84e4755dc4962e3d1093ef100469b15a6fb3f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4cd414ac081fd3a511718e029534c7dbf7f8d0a4

**Pull request merged**